G1ANT.Addon.UI/Commands/UIClickCommand.cs

```diff
@@ -21,11 +24,8 @@ public UIClickCommand(AbstractScripter scripter) : base(scripter)
         public void Execute(Arguments arguments)
         {
             var element = UIElement.FromWPath(arguments.WPath);
-            if (element != null)
-            {
-                element.SetFocus();
-                element.Click();
-            }
+            element?.Click();
+            Wait.UntilInputIsProcessed();
```

Comment:
move this to if (element != null) {} otherwise if element is null you invoke Wait.UntilInputIsProcessed()